examples/api/lib/material/menu_anchor/menu_accelerator_label.0.dart

```diff
@@ -0,0 +1,107 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Flutter code sample for [MenuAcceleratorLabel].
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() => runApp(const MenuAcceleratorApp());
+
+class MyMenuBar extends StatelessWidget {
+  const MyMenuBar({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        Row(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            Expanded(
+              child: MenuBar(
+                children: <Widget>[
+                  SubmenuButton(menuChildren: <Widget>[
+                    MenuItemButton(
+                      child: const MenuAcceleratorLabel('&About'),
+                      onPressed: () {
```

Comment:
What if instead of using MenuAcceleratorCallbackBinding you passed this onPressed callback to MenuAcceleratorLabel as well as MenuItemButton?

Just thinking out loud as I try to understand this.  I'm wondering if the main purpose of MenuAcceleratorCallbackBinding is to avoid passing the callback twice here and to make the API cleaner.